twitcher/models/__init__.py

```diff
@@ -5,8 +5,9 @@
 
 # import or define all models here to ensure they are attached to the
 # Base.metadata prior to any initialization routines
-from .token import AccessToken  # flake8: noqa
 from .service import Service  # flake8: noqa
+from .oauth import Client  # flake8: noqa
+from .oauth import Token  # flake8: noqa
```

Comment:
*to test* (maybe doesn't work for flake)
Usually I write `# noqa: <pep8-code>` which makes it more cross-platform/package/IDE.

Reply:
fixed.